cmd/guacone/cmd/certifier.go

```diff
@@ -52,6 +53,12 @@ var certifierCmd = &cobra.Command{
 			os.Exit(1)
 		}
 
+		err = certify.RegisterCertifier(osv.NewOSVCertificationParser, certifier.CertifierOSV)
```

Comment:
This could be 
``` go
		if err := certify.RegisterCertifier(osv.NewOSVCertificationParser, certifier.CertifierOSV); err != nil {
			logger.Fatalf("unable to register certifier: %w", err)
		}
```